fitbenchmarking/controllers/ralfit_controller.py

```diff
@@ -151,7 +151,8 @@ def hes_eval(self, params, r):
         :param r: residuals, required by RALFit to
                   be passed for hessian evaluation
         :type r: numpy array
-        :return: hessian 2nd order term: sum_{i=1}^m r_i \nabla^2 r_i
+        :return: hessian 2nd order term
+                 sum_{i=1}^m r_i abla^2 r_i
```

Comment:
```suggestion
        :return: hessian 2nd order term
                 :math:`\sum_{i=1}^m r_i \nabla^2 r_i`
```

Reply:
This will lead to a warning.

Reply:
I'm not sure what you mean here. The `\nabla` is a latex symbol so the `\n` is important and I thought that's how to indicate math mode? Maybe you mean it needs to be this?
```
        :return: hessian 2nd order term
                 :math:`\sum_{i=1}^m r_i \nabla^2 r_i`.
```

Reply:
I think I commented before you pushed. Happy with how it looks in the last commit 👍